src/fields/multi_date_picker_field.js

```diff
@@ -69,6 +91,10 @@ export const MultiDatePickerField = ({
         disabled={disabled}
         hideKeyboardShortcutsPanel={hideKeyboardShortcutsPanel}
         numberOfMonths={numberOfMonths}
+        renderMonthElement={monthElement}
+        small={true}
+        showClearDates={showClearDates}
```

Comment:
if you are simply passing it on, i'm fairly sure that the `{...props}` will handle this